Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/SubscriptionsModel.cs

```diff
@@ -52,7 +52,14 @@ public IEnumerable<ISubscriptionHandler> GetHandlers(ModifiedFileModel modifiedF
             {
                 try
                 {
-                    ISubscriptionHandler subscriptionHandler = resolver.Resolve(subscription);
+                    Dictionary<string, string> triggerSubs = new Dictionary<string, string>()
```

Comment:
(minor) What do you think about renaming `triggerSubs` to `parameterSubstitutions`?  I was a little thrown off by this variable name.